### ISSUE TITLE
error while train

### ISSUE THREAD

**Issue by 123liluky** [imatge-upc/rsis#3]
Hi! Thanks for your work.
After installing necessary packages and setting training images,  I run train_leaves.sh, error happened like below:
![1535169193 1](https://user-images.githubusercontent.com/20832778/44614804-b1032b80-a85d-11e8-991e-d3932cdac6df.png)
Do you know the reason? Thank you very much!

**Comment by 123liluky** [comment]
@miriambellver  I revised self.hidden_size/2 to int(self.hidden_size/2), the error was solved. Could you help me explaining few parameters in args.py? The parameters are maxseqlen gt_maxseqlen and best_val_loss. Besides, If there is only one kind of object in my training set, should I set num_classes to 1 or 2? Thank you.

**Comment by miriambellver** [comment]
Hi!

maxseqlen is the sequence length, gt_maxseqlen is how many objects in the ground truth will you consider for the hungarian loss, and best_val_loss is just to track which is the best validation loss. If your dataset has a single object, num_classes should be 2.

Regards,

Míriam
